src/hooks/api/organizers.ts

```diff
@@ -165,6 +166,38 @@ const getOrganizersByCreator = async ({
   return await res.json();
 };
 
+type UseGetOrganizerPermissionsArguments = ServerSideQueryOptions & {
+  organizerId: string;
+};
+
+const getOrganizerPermissions = async ({ headers, organizerId }) => {
+  const res = await fetchFromApi({
+    path: `/organizers/${organizerId}/permissions`,
+    options: {
+      headers,
+    },
+  });
+  if (isErrorObject(res)) {
+    // eslint-disable-next-line no-console
+    return console.error(res);
+  }
+  return await res.json();
```

Comment:
I'm getting a little wary of seeing those lines around every time 😅 we do this little roundtrip in around 20 locations in the codebase, so I think it could be really useful to add a small helper to do this, something in those veins:

```ts
const handleErrorObject = async (response) =>
    // eslint-disable-next-line no-console
    isErrorObject(response) ? console.error(response) : (await response.json());


const getOrganizerPermissions = async ({ headers, organizerId }) =>
    handleErrorObject(fetchFromApi({
        path: `/organizers/${organizerId}/permissions`,
        options: { headers },
    }));
```